zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2913,6 +2997,56 @@ components:
           type: string
           description: The name of the tenant.
 
+    TenantItem:
+      description: Tenant search response item.
+      type: object
+      properties:
+        key:
```

the convention for keys is `{entityType}Key`, check the naming conventions [here](https://docs.google.com/document/d/1rS989EESojWRrCMLH7Nscy1UppbgOanvzVeOljCc_iA/edit?tab=t.0#heading=h.vy58f5kwk0b4)
```suggestion
        tenantKey:
```